source/opt/ccp_pass.cpp

```diff
@@ -27,6 +27,18 @@
 namespace spvtools {
 namespace opt {
 
+bool CCPPass::IsVaryingValue(uint32_t id) const {
+  return id == std::numeric_limits<uint32_t>::max();
```

Comment:
You should use a named constant value for this, to document and enforce consistency with MarkInstructionVarying.

E.g.   enum { kVaryingValueSentinelId = std::numeric_limits<uint32_t>::max(); };